submissions/Nemooochka/friends-app/js/app.js

```diff
@@ -0,0 +1,35 @@
+import {Cards} from "./cards.js";
+import {Url} from "./url.js";
+import {Filters} from "./filters.js";
+
+const cardsWrapBlock = document.querySelector('.wrap-cards');
+
+const urlData = 'https://randomuser.me/api/?results=20';
+let users;
+let filter;
+
+
+(async () => {
+    try {
+        let response = await fetch(urlData);
+        users = await response.json();
+        users = users.results;
+
+        new Cards(users, cardsWrapBlock);
+        filter = new Filters(users, cardsWrapBlock);
+
+        const url = new Url(users, cardsWrapBlock);
+        url.checkUrl();
+    } catch {
+        cardsWrapBlock.innerHTML = `<div class="error">Ooops, something went wrong. Try to reload <i class="far fa-smile-beam"></i></div>`;
+    }
+})();
+
+document.addEventListener("DOMContentLoaded", function () {
+    new Url().onPopStepUrl();
+
+    document.querySelector('.filter-name').addEventListener('keyup', (event) => filter.filterByName(event));
+    document.querySelector('.sort-by_name').addEventListener('click', (event) => filter.sortByName(event));
```

Comment:
Reduce the number of added event listeners. You can combine these `sortByName` and the `sortByAge` handlers